lib/aidoc/detectIntent.ts

```
@@ -0,0 +1,202 @@
+  "vitals",
+  "imaging",
+  "lifestyle",
+  "tips",
+];
+
+// ---------- utils ----------
+function normalize(s: string): string {
+  return s
+    .toLowerCase()
+    .normalize("NFKD")
+    .replace(/[^\x00-\x7F]/g, "")
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)